cypress/support/pages/home.pageObject.js

```diff
@@ -3,9 +3,21 @@ import PageObject from '../PageObject';
 class HomePageObject extends PageObject {
   url = '/#/';
 
+  visitYourFeed() {
+    cy.visit('/#/my-feed/');
+  }
+  
   get usernameLink() {
     return cy.getByDataCy('username-link');
   }
+
+  get navBar() {
+    return cy.getByDataCy('header-navbar');
```

Comment:
The 'navBar' getter is not descriptive enough. It's recommended to use a more specific name for the getter that reflects its purpose or the element it refers to.